scheduler/src/clients/core-client.ts

```diff
@@ -34,3 +35,117 @@ export async function getLatestEventId (): Promise<number> {
   const response = await httpEvents.get('/latest')
   return response.data.eventId || 0
 }
+
+
+/** CONFIG SYNC RELATED **/
```

Comment:
Replaced the "old" sync to sync with datasources (for execution).
We still need a sync with the pipelines to trigger them. Designed it, so we can have a 1-to-n relation between datasources and pipelines. Encapsulated this sync here in the core-client.